cartridge-cli.lua

```diff
@@ -1249,15 +1249,14 @@ Depends: ${deps}
 
 -- * ------------------- Tmpfiles --------------------
 
-local TMPFILES_CONFIG = [[
-d /var/run/tarantool 0755 tarantool tarantool
-]]
+local TMPFILES_CONFIG = 'd /var/run/tarantool 0755 tarantool tarantool'
 
 -- * ------------------- Dockerfile -------------------
 
-local DOCKERFILE_FROM_DEFAULT = 'FROM centos:8'
+local DOCKERFILE_FROM_DEFAULT = 'FROM centos:8\n'
```

Comment:
Please, consider choosing base image, some users could deploy their projects on centos 7 then it should be more native for them to use centos:7 as base image

Reply:
The result package will be the same anyway.
I sure that we need to support building on different platforms, but it will be done in the separated PR, because this change is very-very big and difficult.

Now, there is only one problem with using centos - `makedir` binary abspath in `ExecStartPre` section of unit file will be specific for centos, and it wouldn't work on some debian distributions. But it can be fixed by using custom `unit-template` and `instantiated-unit-template`.